app/controllers/reviews_controller.rb

```diff
@@ -68,13 +68,13 @@ def destroy
     review.destroy
     redirect_to review.submission
   end
-  
+
  def ask
```

Comment:
Mamy tutaj zamieszanie - w zadaniu #161 Pani Krzysztofik użyła nazwy `ask` do metody, która powinna się nazywać np. `send_review_form` (wysłanie odnośnika do formularza recenzji). Ta nazwa logicznie pasuje do zadania, w którym pytamy o to, czy ktoś sporządzi recenzję. 
Co nie zmienia faktu, że oba listy są oddzielne. Dlatego trzeba utworzyć osobną metodę do realizacji tego zadania. Może się ona nazywać `ask_for_review`.